mne_bids/tests/test_read.py

```diff
@@ -196,18 +196,18 @@ def test_handle_info_reading():
     raw = mne.io.read_raw_fif(raw_fname)
     raw.info['line_freq'] = 60
 
+    # write copy of raw with line freq of 60
     # bids basename and fname
     bids_basename = make_bids_basename(subject='01', session='01',
                                        task='audiovisual', run='01')
     kind = "meg"
     bids_fname = bids_basename + '_{}.fif'.format(kind)
+    write_raw_bids(raw, bids_basename, bids_root, overwrite=True)
```

Comment:
the tests were not updated? any way to check that this works?

Reply:
Hmm I suppose we can write a copy of the file into "derivatives/" and check.